src/target_intacct/statistical_journal.py

```diff
@@ -99,12 +97,12 @@ def build_entry(
     location_ids,
     department_ids,
     object_name,
-    account,
+    account_number_column,
 ):
+    # Get cooresponding amount for current account
+    amount = row["Amount" + account_number_column.replace("AccountNo", "")]
```

Comment:
Is this the same as:
`amount = row[account_number_column.replace("AccountNo", "Amount")]`